Fixed red flag splitting across two lines

michaelspellacy opened the pull request:
Addressed Issue #41 Added white-space: nowrap to .flag_red and also added new properties to link within toc to compensate for nowrap. Tested across several browsers and devices. https://www.screencast.com/t/zAm8KP7CPciC Hope this helps! 

p.s. Love what you are doing here!

tatianamac commented on assets/css/structures/_table-of-content.scss:
```diff
@@ -1,3 +1,9 @@
+
+    a {
+    display: inline-block;
+    margin: 0.5rem 0.75rem;
```
This `margin` seems to be adding some extra spacing around each linked word. Is this visual adjustment necessary for the red flags to not break?

If it's not necessarily, I'd prefer the alignment to be left-aligned without the extra padding.

_Left is original where all text is aligned left_; right is PR where there is extra spacing

<img width="516" alt="Side-by-side of [1] correctly rendered text (without a leading space) and [2] incorrectly rendered text (with a leading space)" src="https://user-images.githubusercontent.com/7484549/74808027-07133b00-533e-11ea-9e7d-c8a30c53a914.png">

ovlb replied:
I think it’s fine to have `margin-left: 0` and go with the slightly indented flag. Which happens with non defined words, too.

tatianamac commented on assets/css/base.scss:
```diff
@@ -419,12 +419,13 @@ th {
-  border-radius: 1rem;
-  padding: 0.45rem 0.65rem;
   margin: 0.25rem 0.75rem;
+  padding: 0.45rem 0.65rem;
```
More a curiosity than an issue if you don't mind explaining (for my own edification): Why did you rearrange these properties? 🤓

michaelspellacy replied:
@tatianamac  Sorry again for delay! Day job got out of hand! I simply reorder them alphabetically out of habit for better readability, but happy to change them back in preference to what you may already have in place throughout CSS.

tatianamac replied:
No sorries. Understood; carry on as you are. I was mostly curious.

tatianamac commented on assets/css/structures/_table-of-content.scss:
```diff
@@ -1,3 +1,9 @@
   break-inside: avoid;
+
+    a {
+    display: inline-block;
```
The `inline-block` here causes an issue with the underline for words that become multiple lines, like OCD:

<img width="310" alt="Obsessive Compulsive Disorder has underline that extends beyond the word" src="https://user-images.githubusercontent.com/7484549/74808755-b0a6fc00-533f-11ea-95f9-1616638e5bf9.png">

tatianamac commented:
@michaelspellacy Are you still able to address these comments? No rush on our end as I imagine you're busy, but want to check in. :)

michaelspellacy commented:
Hi @tatianamac and @ovlb!

Took a poke at revisiting this. You'd think it was easy, but no, it never is. :-) This is what I did:

**eleventy.js** - I really felt that a new span was necessary to get us the hook we needed to balance styles on terms with and without hyperlinks. 

**_flag.scss** - Restored old order. Adjusted margin. Added display: inline-block;

**_table-of-contents.scss** - Included <a> and <span> elements (feel free to add a shared class to these in the future). Added margin-right: 0.75rem since it was removed from left margin of .flag__red

Think it looks much better now. Let me know if any new issues!

tatianamac commented:
@ovlb Since you updated the sass last, do you mind taking a look over this? I'll close it once you give it 🐚 of approval

ovlb commented:
With thanks to @michaelspellacy my sea( shel)l of approval is given. 🐚

michaelspellacy commented:
Thanks gang! Glad I could help! ❤️
